docs/source/merlin_commands.rst

```diff
@@ -322,33 +319,44 @@ task server, run:
 
 .. code:: bash
 
-    $ merlin purge <input.yaml>
+    $ merlin purge <input.yaml> [-f] [--steps <steps>]
 
 This will ask you if you would like to remove the tasks, you can use the
 ``-f`` option if you want to skip this.
 
-.. code:: bash
-
-    $ merlin purge -f <input.yaml>
-
 If you have different queues in your workflow yaml file, you can
 choose which queues are purged by using the ``--steps`` argument and
-giving a space separated list of steps.
+giving a space-delimited list of steps.
 
 .. code:: bash
 
     $ merlin purge <input.yaml> --steps step1 step2
 
 
-Debug Info
-----------
-More information can be output by increasing the logging level
+Status (``merlin status``)
```

Comment:
status has --task_server

Reply:
My reasoning here is same as for `stop-workers`